src/components/Button/Button.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../styles/variables";
+@import "../../styles/mixins";
+
+.button {
+  width: 100px;
+  padding: 10px;
+  font-size: 16px;
+  background: transparent;
+  border: none;
+  position: relative;
+  color: $colorBtnFont;
+  z-index: 1;
+
+  &::after {
+    @include absolute-bottom-right-z;
+    -webkit-transform: translate(10px, 10px);
+    -ms-transform: translate(10px, 10px);
```

Comment:
Mixins should be named with a verb or verb phrase and should be camelCased. The mixin 'absolute-bottom-right-z' seems to be in kebab-case.